users/views.py

```diff
@@ -213,3 +227,25 @@ class CustomPasswordResetConfirmView(PasswordResetConfirmView):
 class CustomPasswordResetCompleteView(PasswordResetCompleteView):
 
     template_name = "accounts/password_reset_complete.html"
+
+
+class ActivateAccountView(TemplateView):
+    extra_context = {"MESSAGES": AccountConfirmationText}
+    template_name = "account_confirmation/confirmation.html"
+
+    def activate(self) -> bool:
+        user_id = force_text(urlsafe_base64_decode(self.kwargs["encoded_user_id"]))
+        user = get_user_or_none(user_id=user_id)
```

Comment:
I would suggest checing if user exist, and `return False` upon `DoesNotExist` exception

Reply:
Also done :)